src/scripts/download-new-songs.ts

```diff
@@ -151,6 +144,24 @@ const convertToOpus = async () => {
     for (const mp3File of mp3Files) {
         await ffmpegOpusJob(mp3File);
     }
+
+    let songs: Array<QueriedSong> = await db.kpopVideos("kpop_videos.app_kpop")
```

Comment:
This exists in 1 other place in this file. Should I make a getter function that takes the db and returns this?